contracts/WalletSimple.sol

```diff
@@ -183,14 +183,7 @@ contract WalletSimple is IERC721Receiver, ERC1155Holder {
 
     // Verify the other signer
     bytes32 operationHash = keccak256(
-      abi.encodePacked(
-        getNetworkId(),
-        toAddress,
-        value,
-        data,
-        expireTime,
-        sequenceId
-      )
+      abi.encode(getNetworkId(), toAddress, value, data, expireTime, sequenceId)
```

Comment:
do we need wp changes for operation hash for this?

Reply:
Yes, we will need SDK and WP changes for this